llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -46,6 +46,7 @@
 #include "llvm/Transforms/Utils/ModuleUtils.h"
 #include <map>
 #include <set>
+#include <unordered_set>
```

Comment:
Why this change? Were we pulling it in from ModuleSummaryIndex.h via some other chain of includes?

Reply:
Yes, we need it for the `LocHashToCallSites` type but we were including it via ModuleSummaryIndex.h.